### PR TITLE
Add support for Razer Base Station Chroma

### LINKED ISSUE #632: Support for Razer Base Station Chroma
Apparently the Base Station Chroma isn't supported by the driver, so I thought I'd share my findings.

This device is weird - I wasn't being able to change the color settings by replaying the USB packets. Apparently, it needs some "magic packets" to be sent before it starts accepting chroma settings, for some reason.

Apart from that, it seems to work like any other synapse 3 / software chroma device.

I don't use Linux but macOS, so I coded a driver for it based on code from this repository, and I'd like to give back to this project though, even though I'm not a Linux developer.

Relevant code is here: https://github.com/FruityRazer/FruityRazer/blob/master/FruityRazer/Core/RazerSoftwareChroma.c#L106 (line 106 up to the end of the file). 

`razer_base_station_pre` sends those USB packets that enable chroma.

There's also some code to support the Huntsman Elite in that file.

Let me know if I can be of assistance in anything. 😄

### PR DESCRIPTION
**Pull request by lah7**
Also known as the Razer Headphone/Headset Stand. The effects, brightness and custom matrix works as expected. The product is similar to the HDK (#1003)

Closes #632.

### CODE REVIEW

File: daemon/openrazer_daemon/hardware/accessory.py [hunk 1]
```diff
@@ -48,3 +48,25 @@ class RazerChromaHDK(_RazerDeviceBrightnessSuspend):
...
+    MATRIX_DIMS = [1, 15]
+    METHODS = ['get_device_type_accessory', 'set_static_effect', 'set_wave_effect', 'set_spectrum_effect',
+               'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row']
+
+    DEVICE_IMAGE = "https://rzrwarranty.s3.amazonaws.com/145dcc47f9f9d33b0bd07b066364704160f45e87b756d690b203decec7d1e87c.png"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": "https://rzrwarranty.s3.amazonaws.com/145dcc47f9f9d33b0bd07b066364704160f45e87b756d690b203decec7d1e87c.png",
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/accessory.py]: `"top_img": DEVICE_IMAGE` and remove the `: None` lines

File: driver/razeraccessory_driver.h [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
 #define USB_DEVICE_ID_RAZER_CHROMA_MUG 0x0f07
 #define USB_DEVICE_ID_RAZER_CHROMA_HDK 0x0F09
+#define USB_DEVICE_ID_RAZER_CHROMA_BASE 0x0F08
```
Comment (z3ntu) [inline driver/razeraccessory_driver.h:16]: Please keep the pid ids sorted

**Comment by lah7** [comment]
Thanks for the review - I've addressed your comments. :+1:

**Review by z3ntu — APPROVED** [review]
LGTM

Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @lah7!

:stuck_out_tongue_winking_eye:

**Pull request merged**